### ISSUE TITLE
Metadescription in Snippet Preview contains inline SVG styles

### ISSUE THREAD

**Issue by imminger** [Yoast/Yoast-SEO-for-TYPO3#249]
* [x] I've read and understood the [contribution guidelines](https://github.com/Yoast/Yoast-SEO-for-TYPO3/blob/master/.github/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

### Please give us a description of what happened.
If a site contains an inline SVG file, the content of the style-tag inside the SVG-tag is shown in 
the meta-description of the snippet preview.

<img width="1375" alt="snippetpreview" src="https://user-images.githubusercontent.com/15834641/53874247-50ba2700-4002-11e9-9fa2-e760985a6f4a.png">

![svg content](https://user-images.githubusercontent.com/15834641/53874246-50ba2700-4002-11e9-97e1-2baed4607b89.png)

### Please describe what you expected to happen and why.
It would be less confusing for editors if the snippet preview would only contain text which is visible in the frontend. The style-tag should be ignored in the snippet preview.

### How can we reproduce this behavior?
1. Paste an SVG inline in the HTML (e.g.. in the template of a content element). The SVG must contain a style-tag.
2. Place content-element at top position of the page in the TYPO3 backend, so its content is shown in the snippet preview.

### Technical info
* TYPO3 version: 9.5.4
* Yoast SEO version: 4.1.0
* Relevant extensions in case of a bug: -

**Comment by haassie** [comment]
This is fixed with the version we released today.

**Comment by imminger** [comment]
I tested it with Yoast 5.0.0 and SVGs styles are still visible for me.

I created a small snippet to reproduce the issue:
[snippet.txt](https://github.com/Yoast/Yoast-SEO-for-TYPO3/files/3127783/snippet.txt)

When the snippet is inserted into the template of a content element (i.e. textpic.html), some parts of it will be visible in the snippet preview. 
<img width="636" alt="Preview" src="https://user-images.githubusercontent.com/15834641/56904144-fb4c4480-6a9d-11e9-935c-5677c8be1de6.png">
